command_uninstall.go

```diff
@@ -37,6 +37,18 @@ func cmdUninstall(c *cli.Context) error {
 }
 
 func uninstallPackage(cmd string) error {
+	Outer:
+	for _, packages := range getCommands() {
+		for _, command := range packages.Commands {
+			for _, alias := range command.Aliases {
+				if alias == cmd {
+					fmt.Fprintln(akamai.App.Writer, fmt.Sprintf("Provided name \"%s\" appears to be an alias for installed package \"%s\".", alias, command.Name))
+					cmd = command.Name
+					break Outer
```

Comment:
I think this should be a `goto` instead, with the label above line 52, assuming of course we're not finishing the loops regardless as per the previous comment. Currently the way this codes reads is `Outer` -> loop -> loop -> loop -> back up to `Outer` -> down to line 52. With a go to it reads much more linearly from top to bottom.